### ISSUE TITLE
Feedback for /versioned_docs/version-0.1.0/migration/installation.md

### ISSUE THREAD

**Issue by dzhan008** [Unity-Technologies/com.unity.multiplayer.docs#213]
I'm currently trying to install MLAPI onto my project (currently using **Unity 2019.4.1f1**). I tried to follow the 4th step in the installation step in **Unity 2019.4+ and 2020.x#** but it does not seem to work. Specifically,

```
Enter the Git URL to the MLAPI release package (below). You can click the Copy option in that codeblock and paste it in the Package Manager.

https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi.git?path=/com.unity.multiplayer.mlapi#release/0.1.0
```

I followed the step to copy the URL and add it to the package manager. However, I ran into this error:
![image](https://user-images.githubusercontent.com/9047851/121274411-46d99b00-c87f-11eb-8c7a-8f78afd1b7b3.png)

```
Cannot perform upm operation: Unable to add package [https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi.git?path=/com.unity.multiplayer.mlapi#release/0.1.0]:
  Cannot checkout repository [https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi.git] on target path [com.unity.multiplayer.mlapi]:
    Error when executing git command. fatal: invalid reference: release/0.1.0
 [NotFound]
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()
```
Upon further inspection I read into how the [Git URL](https://docs.unity3d.com/Manual/upm-git.html) was built. Removing the branch at the end (`#release/0.1.0`) seems to allow me to pull properly, but now I'm getting the latest experimental version. Perhaps the branch name may be an issue here?

**Comment by Briancoughlin** [comment]
Will take a look into this. Thanks for raising it.

**Comment by Briancoughlin** [comment]
So this is caused by https://issuetracker.unity3d.com/issues/package-resolution-error-when-using-a-git-dependency-referencing-an-annotated-tag-in-its-git-url

I will update our docs to inform users of this limitation, and the workaround

**Comment by Briancoughlin** [comment]
Work done in PR #208

**Comment by dzhan008** [comment]
Thanks for updating the docs. What exactly is the workaround you mentioned? I don't see anything aside from the resolution note of the bug being fixed in 2019.4.28f1.
